src/core/qgscadutils.h

```diff
@@ -68,6 +70,13 @@ class CORE_EXPORT QgsCadUtils
     {
       public:
 
+        enum LineExtensionSide
```

Comment:
missing dox

Reply:
Do you think I should also move this enum into qgis.h ?

Reply:
yes, move it to qgis.h (please make it a scoped enum and add Q_ENUM).
But the naming and documentation would deserve to be a bit more generic.